wiki/vanity.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Vanity Addresses
+layout: wiki
+redirect_from:
+  - "/Wiki/Vanity"
+---
+
+# What are vanity addresses?
+Vanity addresses are personalized addresses that are created with the intention to include a phrase or name in the address.
+
+This allows the creation of addresses such as: SDeLtA (used by Delta, truncated)
```

Comment:
I would maybe put a full address here and bold/italicize the prefix so the concept is more clear.